rero/rero-ils-ui
# documents: display all fields on the detail view

Co-Authored-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>

## Why are you opening this PR?

- Add some fields.

## How to test?

- Add data to some fields and check the display on "Header" and "Description" tab

## Code review check list
